Sources/HipMobileUI/ViewModels/Views/ExhibitsOverviewListItemVIewModel.cs

```diff
@@ -121,7 +121,7 @@ public void UpdateDistance(Position position)
 
         public void OpenDetailsView(string id)
         {
-            Navigation.InsertPageBefore(new AppetizerPageViewModel(id), downloadPage);
+            Navigation.InsertPageBefore(new AppetizerViewModel(id), downloadPage);
             Navigation.PopAsync();
```

Comment:
As above

Reply:
These changes are from some previous commit. I just changed the name. for fixing ReSharper build error. 
Need to check who made this and why??